sources/api/corndog/src/main.rs

```diff
@@ -393,4 +567,31 @@ mod test {
         let actual_hugepages = check_for_existing_hugepages(existing_value);
         assert_eq!(actual_hugepages, expected_hugepages);
     }
+
+    #[test]
+    fn test_generate_sysctl_config_dash_prefix() {
```

Comment:
This test failed.

Reply:
Fixed in https://github.com/bottlerocket-os/bottlerocket-core-kit/commit/28da3d0d0c28a0aa74cc433e8b6321bfd2808e3b